target/linux/ath79/dts/qca9563_tplink_tl-wpa8630p-v1.dtsi

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
Yes, these files (qca9563_tplink_tl-wpa8630-v1.dts and qca9563_tplink_tl-wpa8630.dtsi) have been duplicated. I thought about preventing compatibility issues that we may have with the other version (tl-wpa8630, without the "p").

Reply:
If the contents of the `.dtsi` are identical, it will be sufficient to duplicate only the `.dts` and change the `#include` filename accordingly.

Typically, we are trying to minimize redundancy by creating shared .dtsi files for devices with similar configuration, however for this device the overhead of yet another cascade might be worse than redundancy (the range WPA-8630xxx device models seems to be a little cluttered anyway).

Unfortunately, .dts filenames are (currently) derived from the device model string in the Makefile, so we indeed have to create a .dts for each model, even when they are identical (e.g. only differ in Makefile rule, like DAP-1330 vs. DAP-1365 or DIR-842 C1 vs. C3)